src/main/capi/duckdb_value-c.cpp

```diff
@@ -275,3 +275,27 @@ duckdb_value duckdb_create_array_value(duckdb_logical_type type, duckdb_value *v
 	}
 	return WrapValue(array_value);
 }
+
+idx_t duckdb_get_map_size(duckdb_value value) {
+	auto val = UnwrapValue(value);
```

Comment:
Can we add some tests for incorrect usage of this API (i.e. passing in an integer value or similar)?